src/main/java/core/basesyntax/service/impl/SupplyAction.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ActionHandler;
+
+public class SupplyAction implements ActionHandler {
+    public void apply(String fruit, int quantity) {
```

Comment:
The `apply` method in the `ActionHandler` interface is expected to return `void`. However, consider if there's any useful information that could be returned from this operation. If the interface allows for it, returning a result might make the system more flexible and informative.